inc/options/pgcache.php

```diff
@@ -1,486 +1,732 @@
 <?php
 namespace W3TC;
 
-if ( !defined( 'W3TC' ) )
+if ( ! defined( 'W3TC' ) ) {
 	die();
-
+}
 ?>
-<?php include W3TC_INC_DIR . '/options/common/header.php'; ?>
+<?php require W3TC_INC_DIR . '/options/common/header.php'; ?>
 
-<form action="admin.php?page=<?php echo $this->_page; ?>" method="post">
+<form action="admin.php?page=<?php echo esc_attr( $this->_page ); ?>" method="post">
 	<p>
-		<?php echo
-sprintf( __( 'Page caching via %1$s is currently %2$s', 'w3-total-cache' ),
-	'<strong>'.Cache::engine_name( $this->_config->get_string( 'pgcache.engine' ) ).'</strong>',
-	'<span class="w3tc-'.( $pgcache_enabled ? 'enabled">' . __( 'enabled', 'w3-total-cache' ) : 'disabled">' . __( 'disabled', 'w3-total-cache' ) ) . '</span>.'
-);
-?>
+		<?php
+		echo wp_kses(
+			sprintf(
+				// translators: 1 HTML strong tag containing PageCache Engine name, 2 HTML span tag containing PageCache Engine enabled/disabled.
+				__(
+					'Page caching via %1$s is currently %2$s',
+					'w3-total-cache'
+				),
+				'<strong>' . esc_html( Cache::engine_name( $this->_config->get_string( 'pgcache.engine' ) ) ) . '</strong>',
+				'<span class="w3tc-' . ( $pgcache_enabled ? 'enabled">' . __( 'enabled', 'w3-total-cache' ) : 'disabled">' . __( 'disabled', 'w3-total-cache' ) ) . '</span>.'
```

Comment:
Use `esc_html__()`.

Reply:
Switched to esc_html__ for both instances